extensions/wikia/SEOTweaks/scripts/uncrawlable-urls.js

```diff
@@ -6,14 +6,19 @@ require([
 	window
 ) {
 	'use strict';
-	$(function () {
+
+	const decodeUncrawlableURL = function() {
```

Comment:
`const` is a JavaScript ES6 keyword and therefore does not work on older browsers. Since Internet Explorer 11 is a [supported browser](https://c.wikia.com/wiki/Help:Supported_browsers) I believe just:
```javascript
function decodeUncrawlableURL() {
    // ...
}
```
is preferred here.